pkg/util/intl/resolve.go

```diff
@@ -26,3 +26,19 @@ func Resolve(preferred []string, fallback string, supported []string) (int, lang
 
 	return -1, tag
 }
+
+// ResolveUnicodeCldr resolves language tag to Unicode CLDR language tag.
+func ResolveUnicodeCldr(lang language.Tag, fallback language.Tag) string {
+	var cldrTags []language.Tag
+	for _, cldr := range CldrLanguages {
+		cldrTags = append(cldrTags, language.MustParse(cldr))
+	}
```

Comment:
Parse all cldr languages and put them in memory so that we don't have to parse it in each request?
I remember there was a performance issue saying that constructing the matcher is expensive.

Reply:
Yes, I updated to use `matcher := GetMatcher(cldrTags)` in aad899c54f1c24cd4e7641987957ce044b380c53

Reply:
My thought is to construct the matcher in `init()` and store it to a variable, because the list of clrd locales never changes.
Now for each request we looped all items and called `language.MustParse` for each item, which might be expensive?

Reply:
Sure updated